kartothek/io/dask/common_cube.py

```diff
@@ -489,19 +516,23 @@ def _multiplex_store_dataset_from_partitions_flat(
     result = {}
     for k, v in dct.items():
         if update:
+            print(
+                f"_multiplex_store_dataset_from_partitions_flat: {k} delete scope: {delete_scopes.get(k, [])}"
+            )
             ds_factory = metadata_factory_from_dataset(
                 existing_datasets[k], with_schema=True, store=store
             )
             result[k] = update_dataset_from_partitions(
                 v,
                 dataset_uuid=cube.ktk_dataset_uuid(k),
-                delete_scope=[],
+                delete_scope=delete_scopes.get(k, []),
                 ds_factory=ds_factory,
                 metadata=metadata[k],
                 metadata_merger=None,
                 store_factory=store,
             )
         else:
+            print(f"_multiplex_store_dataset_from_partitions_flat: {k} no update")
```

Comment:
same

Reply:
Removed in 0938f6662cd71d3cc18b622ff52f1fd8df9661bd